### ISSUE TITLE
Crashed when trying to download from cartoon.adult-fanfiction.org

### ISSUE THREAD

**Issue by arisboch** [Birion/python-ffdl#28]
Here's the error message:

```
steven@steven-H31 ~/Temp % pyffdl download  https://cartoon.adult-fanfiction.org/story.php\?no\=600094777
Downloading https://cartoon.adult-fanfiction.org/story.php?no=600094777
/home/steven/.local/lib/python3.10/site-packages/ebooklib/epub.py:1395: UserWarning: In the future version we will turn default option ignore_ncx to True.
  warnings.warn('In the future version we will turn default option ignore_ncx to True.')
Traceback (most recent call last):
  File "/usr/local/bin/pyffdl", line 8, in <module>
    sys.exit(cli())
  File "/home/steven/.local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/steven/.local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/steven/.local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/steven/.local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/steven/.local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/steven/.local/lib/python3.10/site-packages/pyffdl/core/app.py", line 86, in cli_download
    download(urls, verbose)
  File "/home/steven/.local/lib/python3.10/site-packages/pyffdl/core/app.py", line 50, in download
    story.run()
  File "/home/steven/.local/lib/python3.10/site-packages/pyffdl/sites/story.py", line 272, in run
    self.make_title_page()
  File "/home/steven/.local/lib/python3.10/site-packages/pyffdl/sites/aff.py", line 72, in make_title_page
    _header = self.page("table")[2].table("td")
IndexError: list index out of range

```

Is there a way to solve this problem?

**Comment by Birion** [comment]
AFF completely reworked their formatting, which then borked everything up. It should be fixed now in the 0.20.0 release.

**Comment by arisboch** [comment]
@Birion Thanks!
